.github/workflows/merge-to-stage.js

```diff
@@ -217,6 +217,26 @@ const main = async (params) => {
     if (stageToMainPR) body = stageToMainPR.body;
     existingPRCount = body.match(/https:\/\/github\.com\/adobecom\/milo\/pull\/\d+/g)?.length || 0;
     console.log(`Number of PRs already in the batch: ${existingPRCount}`);
+    
+    if (params.context.eventName === 'pull_request_target' && params.context.payload.pull_request.merged) {
+      const mergedPR = params.context.payload.pull_request;
+      if (!body.includes(mergedPR.html_url)) {
+        body = `- ${mergedPR.html_url}\n${body}`;
+        if (stageToMainPR) {
+          console.log("Updating PR's body with merged PR...");
+          await github.rest.pulls.update({
+            owner,
+            repo,
+            pull_number: stageToMainPR.number,
+            body,
+          });
+          console.log(`Updated Stage to Main PR #${stageToMainPR.number} with merged PR #${mergedPR.number}`);
+        } else {
+          await openStageToMainPR();
+        }
+      }
+      return;
+    }
```

Comment:
We already do this in this file, here: https://github.com/adobecom/milo/blob/stage/.github/workflows/merge-to-stage.js#L229-L239

The idea is to have this similar to the slack webhook you contributed and work outside of the `merge-to-stage` flow, but whenever something is manually merged as well